images/build-drivers/build-drivers.sh

```diff
@@ -27,6 +27,13 @@ function start_docker() {
 }
 
 PUBLISH_S3="${PUBLISH_S3:-false}"
+
+TARGET_DISTRO="${1}"
+TARGET_KERNEL="${2}"
+DBG_ENV_PARAMS="-e TARGET_DISTRO=${TARGET_DISTRO}"
```

Comment:
There's any reason to do not check the `TARGET_DISTRO` value?

Reply:
I don't think so. I wrongly leaved it with the assumption that is always passed. Going to fix it

Reply:
I did a refactoring @leodido on `build-drivers.sh` script. So check each argument in a way that we can also accept a third to specify the `TARGET_VERSION` (the version of the Falco driver). Let me know WDYT, so that in case I resolve this review.

In the end the script can now be called like that:
```sh
PUBLISH_S3=false ENSURE_DOCKER=false DBG_WORKDIR=$PWD/driverkit \
  ./images/build-drivers/build-drivers.sh amazonlinux 4.14.77 85c88952b018fdbce2464222c3303229f5bfcfad
```

Reply:
(Also, added the jobs for Linux kernel 3.*)